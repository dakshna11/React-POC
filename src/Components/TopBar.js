import React from 'react';
function TopBar() {
    return (
        <div className="topbar">
            <div><span id="logo-1">Auto</span><span id="logo-2">Wash</span></div>

            <div>
                <div class="top-bar-icon">
                    <i class="far fa-clock"></i>
                </div>
                <h3>Opening Hour</h3>
                <p>Mon - Fri, 8.00 - 9.00</p>
            </div>
             
            <div><h3>Call Us</h3><p>+012 345 6789</p></div>
            <div><h3>Email Us</h3><p>info@example.com</p></div>
        </div>
        
    );
}

export default TopBar;