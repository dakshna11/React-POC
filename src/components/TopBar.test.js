import React from "react";
import { shallow } from "enzyme";
import TopBar from "./TopBar";

const topBar = 'Test TopBar'
let wrapped = shallow(<TopBar>{topBar}</TopBar>)

describe('TopBar', ()=>{
    it('should render the TopBar Component correctly', ()=>{
        expect(wrapped).toMatchSnapshot()
    })
})