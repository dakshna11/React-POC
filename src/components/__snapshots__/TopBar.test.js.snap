// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopBar should render the TopBar Component correctly 1`] = `
<div
  className="topbar"
>
  <div
    style={
      Object {
        "marginLeft": "-6%",
        "marginTop": "1%",
      }
    }
  >
    <span
      id="logo-1"
    >
      Auto
    </span>
    <span
      id="logo-2"
    >
      Wash
    </span>
  </div>
  <div
    className="topbar-main"
  >
    <div
      className="topbar-content-main"
    >
      <img
        height="35px"
        src="clock.png"
        style={
          Object {
            "marginTop": "20%",
          }
        }
        width="35px"
      />
      <div
        className="topbar-content"
      >
        <h3
          style={
            Object {
              "marginBottom": "0px",
            }
          }
        >
          Opening Hour
        </h3>
        <p
          style={
            Object {
              "width": "150px",
            }
          }
        >
          Mon - Fri, 8.00 - 9.00
        </p>
      </div>
    </div>
    <div
      className="topbar-content-main"
    >
      <img
        height="35px"
        src="phone-call.png"
        style={
          Object {
            "marginTop": "20%",
          }
        }
        width="35px"
      />
      <div
        className="topbar-content"
      >
        <h3
          style={
            Object {
              "marginBottom": "0px",
            }
          }
        >
          Call Us
        </h3>
        <p
          style={
            Object {
              "width": "150px",
            }
          }
        >
          +012 345 6789
        </p>
      </div>
    </div>
    <div
      className="topbar-content-main"
      style={
        Object {
          "marginLeft": "2%",
        }
      }
    >
      <img
        height="35px"
        src="email.png"
        style={
          Object {
            "marginTop": "20%",
          }
        }
        width="35px"
      />
      <div
        className="topbar-content"
      >
        <h3
          style={
            Object {
              "marginBottom": "0px",
            }
          }
        >
          Email Us
        </h3>
        <p
          style={
            Object {
              "width": "150px",
            }
          }
        >
          info@example.com
        </p>
      </div>
    </div>
  </div>
</div>
`;
